roles/horizon/tasks/main.yml

```diff
@@ -173,6 +173,42 @@
     - horizon.favicon_url is defined
     - openstack_install_method == 'distro'
 
+- name: rcue settings file exists (rhel osp)
```

Comment:
Can we add below 4 task in a block with when: - openstack_install_method == 'distro'